espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/Element.kt

```diff
@@ -137,3 +143,39 @@ class Element(view: View) {
         }
     }
 }
+
+class ComposeElement(node: SemanticsNodeInteraction) : BaseElement {
+    @Suppress("JoinDeclarationAndAssignment")
+    @SerializedName(JSONWP_ELEMENT_KEY, alternate = [W3C_ELEMENT_KEY])
+    override val element: String
+
+    init {
+        element = UUID.randomUUID().toString()
+        ComposeViewCache.put(element, node)
+    }
+
+    companion object {
```

Comment:
I would rather move this helper into a separate module and made it to a global function. It is usually not the best approach to put logic into model dclarations

Reply:
moved